src/app/workflow/launch-third-party/dialog/launch-to-codespace-dialog.component.html

```diff
@@ -0,0 +1,38 @@
+<h1 mat-dialog-title>Launch this {{ term }} to a GitHub Codespace?</h1>
+<div mat-dialog-content>
+  <app-alert></app-alert>
+  <p>
+    You are about to launch the {{ term }} <b>{{ path }}</b> in a GitHub Codespace. To configure the Codespace, GitHub reads a
+    <a href="https://containers.dev" target="_blank" rel="noopener noreferrer">development container</a> ("dev container") file from the
+    repository, which specifies the files to use and software to run.
+  </p>
+  <p *ngIf="!hasDevcontainer">
+    <b>No dev container files were detected in this version.</b> When GitHub can't find a dev container file, it creates a default Codespace
+    environment that may not automatically run the {{ term }} or the software required to support it.
+  </p>
+  <p *ngIf="hasDevcontainer">
+    This version contains dev container files, but Dockstore could not confirm that they will automatically run the {{ term }} when the
+    Codespace starts.
+  </p>
+  <p>
+    After the Codespace UI loads, to run the {{ term }},
+    <b>you may need to click on the {{ term }} file and select the appropriate menu options to enable the required extensions.</b>
+  </p>
+  <p>
+    If you own the repository, you can create a dev container file that will automatically run this {{ term }}. See
+    <a
+      href="{{ Dockstore.DOCUMENTATION_URL }}/getting-started/getting-started-with-notebooks.html#launch-with-github-codespaces"
+      target="_blank"
+      rel="noopener noreferrer"
+      >our documentation</a
+    >
+    for more information.
+  </p>
+  <p>Launch this {{ term }} to a GitHub Codespace?</p>
+  <div class="btn-group" fxLayout="row" fxLayoutAlign="end center" fxLayoutGap="1rem">
+    <button mat-flat-button (click)="no()" data-cy="archive-no" [disabled]="clicked" mat-dialog-close class="secondary-1">Cancel</button>
+    <button mat-flat-button (click)="yes()" data-cy="archive-yes" [disabled]="clicked" color="warn">
```

Comment:
I think we usually use a blue button for an action that isn't "destructive", like below. The class for the export button in the screen shot is `accent-1-dark-btn`

![image](https://github.com/dockstore/dockstore-ui2/assets/25287123/dc09122d-f444-45d9-9bf9-d41fd73a93f5)